README.md

```diff
@@ -50,6 +50,13 @@ You can install the package via composer:
 composer require spatie/schema-org
 ```
 
+### Supported Schema Versions
+| Package Version | Schema Version |
+|-----------------|----------------|
+| 3.x.x           | V9             |
+| 4.x.x           | V10            |
+
+Note: Only lists _current_ versions of the spatie/schema-org package.
```

Comment:
Hey, good idea but please remove it. I know that this list won't be maintained and we also only maintain the `main` branch. So all older releases are only there for history completeness - which version matches which schema version can also be checked in the changelog.

Reply:
Ahh - good deal. Wasn't sure if any updates/fixes to these "older" releases would matter.

Will get that removed now.